src/app/shared/navbar/navbar.component.ts

```diff
@@ -46,7 +47,7 @@ import {
   ]
 })
 export class NavbarComponent implements OnInit, AfterContentChecked, OnDestroy {
-  // FIXME: Convert 'class' to 'type' for consistancy
+  // FIXME: Convert 'class' to 'type' for consistency
```

Comment:
What is the context of this FIXME? Is it saying to convert the `NavbarComponent` class to a type? I'm not really following.

Reply:
This FIXME was was there when I found it, I have no context. I just saw the typo and fixed it. It doesn't appear to apply to anything in the NavbarComponent (since that class cannot be converted to a type) so my guess is it can be removed.

Reply:
I'm going to guess if we look at the git history for this file it was referring to something else that got removed and it didn't. My vote would be 🔥 it